actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
```

Comment:
That was something I was wondering about. The main argument against it is these methods (`content_tag_string`, `tag_options`) are shared between legacy and modern syntax. I means that `tag` and `content_tag` methods would need builder instance to use them. Sounds event weirder to me. What do you think?